pkg/services/object/put/distributed.go

```diff
@@ -127,6 +130,17 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 		}
 	}
 
+	var deletedObjs []oid.ID
+	var lockedObjs []oid.ID
+	switch t.objMeta.Type() {
+	case objectSDK.TypeTombstone:
+		deletedObjs = t.objMeta.Objects()
+	case objectSDK.TypeLock:
+		lockedObjs = t.objMeta.Objects()
+	default:
+	}
+
+	t.objSharedMeta = object.MetaInfo(t.obj.GetContainerID(), t.obj.GetID(), t.obj.PayloadSize(), deletedObjs, lockedObjs, t.obj.CreationEpoch())
```

Comment:
`t.objSharedMeta` is used conditionally while calculated unconditionally. Lets sync

Reply:
can you elaborate a little? this is calculated before any concurrent code, but is read concurrently, yes. do you mean some comments about it? or you want to see a lock here?